db/view/view.cc

```diff
@@ -88,7 +89,7 @@ view_info::view_info(const schema& schema, const raw_view_info& raw_view_info, s
 view_info::view_info(const schema& schema, const raw_view_info& raw_view_info, db::view::base_dependent_view_info base_info)
         : _schema(schema)
         , _raw(raw_view_info)
-        , _base_info(make_lw_shared<const db::view::base_dependent_view_info>(base_info))
+        , _base_info(base_info)
```

Comment:
Maybe I'm missing something, but why are we copying here instead of moving?

Reply:
I changed it to a move in the rebase (although in this case they are equally expensive - base_info is now just a struct of 3 bools)